pyproject.toml

```diff
@@ -26,7 +32,7 @@ python = "^3.7.2"
 email-validator = "^1.1.3"
 pydantic = "^1.9.0"
 python-dotenv = "^0.20.0"
```

Comment:
i wonder if we should also loosen these constraints as well for the same reasons
```suggestion
email-validator = "^1.1"
pydantic = "^1.8"
python-dotenv = "^0.11"
```

thoughts @tylerscheuble @vvillait88?

Reply:
Skimmed through the changelogs and found no obvious red flags, but we can update the requirements and run tests to confirm.

Also @AustinSeftonPDL you'll need to run `poetry lock` to update the lock file as well.

Reply:
@martimlobao Do we need to update requirements.txt or just drop that file as it is not used anywhere?

Reply:
Yes, not sure what to do with `requirements.txt`.